.taskcluster.yml

```diff
@@ -33,380 +33,342 @@ tasks:
-          command:
-            - "/bin/sh"
-            - "-lcxe"
-            - "git clone --quiet ${repository} /bugbug &&
-              cd /bugbug &&
-              git -c advice.detachedHead=false checkout ${head_rev} &&
-              cd ui/changes &&
-              npm install --no-progress &&
-              npm run release"
-          artifacts:
-            public/frontend:
-              expires: { $fromNow: "2 weeks" }
-              path: /bugbug/ui/changes/dist
-              type: directory
-        metadata:
-          name: bugbug ui build
-          description: bugbug ui build
-          owner: mcastelluccio@mozilla.com
-          source: ${repository}/raw/${head_rev}/.taskcluster.yml
+        - $mergeDeep:
+            - { $eval: default_task_definition }
+            - taskId: { $eval: as_slugid("frontend_build") }
+              payload:
+                image: node:lts
+                command:
+                  - "/bin/sh"
+                  - "-lcxe"
+                  - "git clone --quiet ${repository} /bugbug &&
+                    cd /bugbug &&
+                    git -c advice.detachedHead=false checkout ${head_rev} &&
+                    cd ui/changes &&
+                    npm install --no-progress &&
+                    npm run release"
+                artifacts:
+                  public/frontend:
+                    expires: { $fromNow: "2 weeks" }
+                    path: /bugbug/ui/changes/dist
+                    type: directory
+              metadata:
+                name: bugbug ui build
+                description: bugbug ui build
 
-      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
-        then:
-          dependencies:
-            - { $eval: as_slugid("lint_task") }
-            - { $eval: as_slugid("tests_task") }
-            - { $eval: as_slugid("http_tests_task") }
-            - { $eval: as_slugid("frontend_build") }
-            - { $eval: as_slugid("packaging_test_task") }
-            - { $eval: as_slugid("version_check_task") }
-            - { $eval: as_slugid("integration_test") }
-          scopes:
-            - secrets:get:project/bugbug/deploy
-          created: { $fromNow: "" }
-          deadline: { $fromNow: "1 hour" }
-          provisionerId: proj-bugbug
-          workerType: batch
-          payload:
-            features:
-              taskclusterProxy: true
-            maxRunTime: 3600
-            image: "${taskboot_image}"
-            command:
-              - taskboot
-              - deploy-pypi
-            env:
-              TASKCLUSTER_SECRET: project/bugbug/deploy
-              GIT_REPOSITORY: "${repository}"
-              GIT_REVISION: "${head_rev}"
-          metadata:
-            name: bugbug PyPI release
-            description: bugbug PyPI release
-            owner: mcastelluccio@mozilla.com
-            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+        # why does this not have a task id ?
```

Comment:
why does this task not have a task id ?

Reply:
Nice catch!

I'm not sure; perhaps it was a mistake. Either way, this is outside the scope of this PR. Could you please create an issue for this and remove the comment from the `.taskcluster.yml` file?

Reply:
comment removed

Reply:
Thank you!